src/modules/Dither/info.json

```diff
@@ -2,12 +2,18 @@
     "name": "Dither",
     "description": "Approximates a color from a mixture of other colors when the required color is not available, creating illusions of the color that is not present actually.<a href='https://en.wikipedia.org/wiki/Dither'>Read more</a>",
     "inputs": {
+	"channel": {
       "dither": {
         "type": "select",
         "desc": "Name of the Dithering Algorithm",
         "default": "none",
-        "values": ["none","floydsteinberg","bayer","Atkinson"]
+        "values": ["none","floydsteinberg","bayer","Atkinson"]{
       }
     }
+},
+	"docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#dither-module"
+  },
+	"wikipedia-link":"https://en.wikipedia.org/wiki/Dither"
```

Comment:
Don't add any Wikipedia links here. Add the links to the MODULES.md file near the module names(for more info on it). Don't add Wikipedia links to all modules, only for dither is enough.